ros2/source/Makefile

```diff
@@ -10,10 +10,13 @@ help:
 	@echo ""
 
 build:
+	@docker build --tag=ros2:devel			devel/.
```

Comment:
Would you want the `source` image to build FROM the `devel`? This Could reduce duplication.

Reply:
yeah that was my original idea. my understanding from https://github.com/osrf/docker_templates/pull/49#issuecomment-474635769 was that you liked the idea of using the same template from both.

But we can go the other way and reduce the source template to be based on devel and just clone, build and source the workspace

Reply:
Using nested tags may complicate automated builds given the pricipaled ordering of builds, but given we aren't doing that with `source`, and only the `nightly` tag is built via osrf's automated repo on Dockerhub, I think we are fine in separating the devel setup and the source compilation into separate Dockerfiles.

Reply:
:+1: please see https://github.com/osrf/docker_templates/pull/57 for a version with nested templates